examples/firmware_swapping/Makefile

```diff
@@ -0,0 +1,66 @@
+# If no BOARD is found in the environment, use this default:
+export BOARD ?= iotlab-m3
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Activate FW slots
+FW_SLOTS = 1
+
+FW_METADATA_SPACE = 0x100  # 76 bytes meta-data, 256 byte aligned
+
+# Define the parameters for the FW slot 1
+FW_IMAGE1_OFFSET = 0x08004000   # Start at page 8
+FW_IMAGE1_LENGTH = 0x3C000  # Reserve 120 pages
+FW_IMAGE1_END = 0x08040000 # FW_IMAGE_OFFSET + FW_IMAGE_LENGTH
+VERSION_IMG1 = 0x1
+UUID_IMG1 = 0xabcd1234
+
+# Define the parameters for the FW slot 1
+FW_IMAGE2_OFFSET = 0x08040000   # Start at page 128
+FW_IMAGE2_LENGTH = 0x3C000  # Reserve 120 pages
+FW_IMAGE1_END = 0x0807C000 # FW_IMAGE_OFFSET + FW_IMAGE_LENGTH
+VERSION_IMG2 = 0x2
+UUID_IMG2 = 0xabcd1234
+
+bootloader:
+	@cd ../bootloader; \
+	CFLAGS+=-DFW_METADATA_SPACE=$(FW_METADATA_SPACE) make clean all; \
+	cp bin/$(BOARD)/bootloader.hex ../firmware_swapping
+
+gnrc_networking-slot1:
```

Comment:
Maybe is more intuitive to to `make gnrc_networking SLOT=1` than to call `make gnrc_networking-slot1`.